content.js

```diff
@@ -305,7 +305,10 @@ if (window.contentScriptInjected !== true) {
         '.net',
         '.info',
         '.rs',
-        '.org'
+        '.org',
+        '©',
+        '®',
+        '\u2122'
```

Comment:
Why did you use Unicode codepoint instead of actual character here? 

If you decide to leave this as it is, can you add a comment describing which character are you referring to?

Reply:
Unicode was here by mistake.